src/services/service-dogs.js

```diff
@@ -15,3 +15,23 @@ export async function getDogsById(id) {
   const resp = await client.from('dogs').select(`*`).match({ id }).single();
   return checkError(resp);
 }
+
+export async function updateDog(dog) {
+  const resp = await client.from('dogs').update(dog).eq('id', dog.id);
+  return checkError(resp);
+}
+
+export async function createDog(dog) {
+  const resp = await client.from('dogs').insert({
```

Comment:
There's no image here. Try just insert({ dog})

Reply:
So I should just take out the key values and put in the object?